velox/exec/tests/HashJoinTest.cpp

```diff
@@ -638,6 +638,20 @@ class HashJoinBuilder {
         memory::spillMemoryPool()->stats().peakBytes;
     TestScopedSpillInjection scopedSpillInjection(spillPct);
     auto task = builder.assertResults(referenceQuery_);
+    // Wait up to 5 seconds for all the task background activities to complete.
```

Comment:
Would that also cause stats to be missing in production? There is no special wait like this there.